MessengerProj/src/main/res/values/strings.xml

```diff
@@ -304,107 +304,107 @@
     the string needed for video time+size (0:00, 12,3 Mib) -->
-    <string name="OneMoment">One moment …</string>
-    <string name="NoChatsHelp">Start messaging by pressing the new chat button in the bottom right corner or tap the menu button for more options.</string>
+    <string name="OneMoment">One moment…</string>
+    <string name="NoChatsHelp">Start messaging by pressing the \"New chat"\ button in the bottom right, or tap the \"Menu"\ button for more options.</string>
     <string name="Intro7Message"><![CDATA[<b>Delta Chat</b>]]> is safe for business use, compatible and standards-based.</string>
     <string name="InviteMenuEntry">Send invitations</string>
-    <string name="InviteText">I\'m using Delta Chat messenger now - %1$s - you can text me at %2$s</string>
+    <string name="InviteText">Shhh, Delta Chat messenger now - %1$s - you can text me at %2$s</string>
     <string name="AdvancedSettings">Advanced settings</string>
-    <string name="AskResetNotifications" >Reset all notification settings and sounds of this page and of your contacts and groups?</string>
+    <string name="AskResetNotifications" >Reset all notification settings and sounds of current page, contacts and groups?</string>
     <string name="AttachFiles">Attach files</string>
     <string name="ErrGroupNameEmpty">Please enter a name for the group.</string>
-    <string name="MsgNewGroupDraftHint">Write a first message to allow others to reply in this group.\n\n• It is okay if not all members use Delta Chat.\n\n• Delivering the first message may take a moment.</string>
+    <string name="MsgNewGroupDraftHint">Compose the first message, allowing others to reply within this group.\n\n• It is okay if not all members use Delta Chat.\n\n• Delivering the first message may take a while.</string>
     <string name="MsgNewGroupDraft">Hello, I\'ve just created the group \"%1$s\" for us.</string>
     <string name="MsgGroupNameChanged">Group name changed from \"%1$s\" to \"%2$s\".</string>
     <string name="MsgGroupImageChanged">Group image changed.</string>
     <string name="MsgMemberAddedToGroup">Member %1$s added.</string>
     <string name="MsgMemberRemovedFromToGroup">Member %1$s removed.</string>
     <string name="AskAddMemberToGroup">Add <![CDATA[<b>]]>%1$s<![CDATA[</b>]]> to group?</string>
     <string name="AskRemoveMemberFromGroup">Remove <![CDATA[<b>]]>%1$s<![CDATA[</b>]]> from group?</string>
-    <string name="ErrSelfNotInGroup">You must be a member of the group to perform this action.</string>
-    <string name="MsgGroupLeft">Group left.</string>
-    <string name="NoMessagesHint">Send message to <![CDATA[<b>]]>%1$s<![CDATA[</b>]]>:\n\n• It is okay if  <![CDATA[<b>]]>%2$s<![CDATA[</b>]]> does not use Delta Chat.\n\n• Delivering the first message may take a moment.</string>
+    <string name="ErrSelfNotInGroup">You must be a group member to perform this action.</string>
+    <string name="MsgGroupLeft">Group disbanded.</string>
```

Comment:
well, it's only a single member that has left the group, the group itself stays intact and unchanged.

Reply:
What is the context here?

Reply:
MsgGroupLeft ist shown as in place of the message (in the bubble) text a user has left a group by himself.
It's a sort of a system message, however, as the action comes from the user, it is shown in its name. (if someone finds some time, however, it might be a good idea show these messages eg. in italics)

Reply:
My opinion is to show a full sentence: 

- "I have left the group!" or
- "I have left this group!"

Reply:
this "I have" implies the message was written by the user in this form. I do not think, this is a good idea. I would prefer a more neutral message (which could be formatted as a clear "system message" then, if in doubt, just using eg. italics)

Reply:
Hmmm, bubble? In my test scenario this text was shown as a real message.
The user did not write the text (in this form), but he triggered it by pressing "the button" and telling the others "I quit".

It was even shown in the users (the user who left) chat as outgoing text.
So, I tought it makes more sense to use the "I-form" (strange word).

Reply:
we're talking about the same stuff. Currently, the message is shown as if it was typed in this form by the user - however, this was only to keep things simple, this is not my desired form. I would like the "status messages" coming from a user - but not typed by him - to be clearly distinguishable from normal text messages - eg. by using italics and a neutral speech.

Reply:
Well, what if it isn't an active decision on part of the user (?), stemming instead from something going wrong, closing the chat as a result.